django/views/generic/edit.py

```diff
@@ -53,7 +53,7 @@ def get_form_kwargs(self):
             'prefix': self.get_prefix(),
         }
 
-        if self.request.method in ('POST', 'PUT'):
+        if self.request.method in ('POST', 'PUT', 'DELETE'):
```

Comment:
about add `'DELETE'` http method in `FormMixin.get_form_kwargs`:
I'm not sure if it's good to change the behavior of FormMixin, just to accept the use case of DeletionMixin or BaseDeleteView. Changing the existing API may end up causing unexpected problems.

Perhaps modifications should be made in DeletionMixin or BaseDeleteView